### PR TITLE
fix unable to restrict / ban users not participant of the group

### PR DESCRIPTION
**Pull request by SpEcHiDe**
follow up of #395

### CODE REVIEW

File: userge/plugins/admin/gadmin.py [hunk 1]
```diff
@@ -204,16 +204,13 @@ async def kick_usr(message: Message):
     if not user_id:
         return await message.err("no valid user_id or message specified")
     try:
-        get_mem = await chat.get_member(user_id)
-        await chat.kick_member(user_id)
+        get_mem = await message.client.get_users(user_id)
+        await chat.kick_member(user_id, until_date=time.time() + 39)
```
Comment (rking32) [inline userge/plugins/admin/gadmin.py:208]: `until_date` works fine here? @tomyprs said that it's not working as we expect.

Although, this arg feels like useless if chat peer isn't an instance of `raw.types.InputPeerChannel`
https://github.com/pyrogram/pyrogram/blob/a460d12a824ff61f1c96842b13f1fdd2ab0a7419/pyrogram/methods/chats/kick_chat_member.py#L93-L98

**Review by rking32 — COMMENTED** [review]
Thanks again for your contribution :)

File: userge/plugins/admin/gadmin.py (hunk 1 above)
Comment (SpEcHiDe) [reply userge/plugins/admin/gadmin.py:208]: oops, sorry. 

I was only considering Supergroups and Channels.

Forgot about basic groups. 😣

Wait 🤧

Comment (tomyprs) [reply userge/plugins/admin/gadmin.py:208]: @rking32  Sometimes it's not working properly 🤔.

Comment (rking32) [reply userge/plugins/admin/gadmin.py:208]: @SpEcHiDe how about this?

if chat type is group -> tomy's method
else -> your's method (channel or super group)

**Comment by rking32** [comment]
Can we merge this?

**Comment by tomyprs** [comment]
I think why should we restrict someone who isn't even in the group ? 🤔

**Comment by SpEcHiDe** [comment]
example scenario:
you use userge, 
you are an admin of a group with a linked channel
some user is spamming through the comments
you try to use mute command

expected result (atleast how I want it to be 😅)
the user is muted

actual result (before this pr)
userge shows the error message

**Pull request merged**